tools/formatter/Dockerfile

```diff
@@ -0,0 +1,70 @@
+# Copyright 2021 The MLPerf Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+# This Docker image is used to run code formatter
+
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND="noninteractive"
+
+# Install dev packages
+RUN apt-get update && apt-get upgrade -y && apt-get install -y --no-install-recommends \
+    tzdata gcc make git curl wget zip unzip tar gzip
+
+# Set timezone to UTC by default
+RUN ln -sf /usr/share/zoneinfo/Etc/UTC /etc/localtime
+
+ENV FORMATTER_HOME /opt/formatters/bin
+ENV PATH=${PATH}:${FORMATTER_HOME}
+RUN mkdir -p ${FORMATTER_HOME}
+
+# Add Java formatter
+RUN apt-get install -y openjdk-11-jdk-headless
+RUN curl --output `dirname ${FORMATTER_HOME}`/google-java-format-1.9-all-deps.jar \
+         -L https://github.com/google/google-java-format/releases/download/google-java-format-1.9/google-java-format-1.9-all-deps.jar
+ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-amd64
```

Comment:
Is this really needed?
I believe that `apt-get install -y openjdk-11-jdk-headless` command should set this automatically.

Reply:
Fixed